metadata.json

```diff
@@ -33,5 +33,48 @@
       "version_requirement": ">= 7.0.0"
     }
   ],
-  "data_provider": null
+  "operatingsystem_support": [
+    {
+      "operatingsystem": "CentOS",
+      "operatingsystemrelease": [
+        "7",
+	"8"
+      ]
+    },
+    {
+      "operatingsystem": "OracleLinux",
+      "operatingsystemrelease": [
+        "7",
+	"8"
```

Comment:
See above, while indentation for RedHat seems correct.

Reply:
Same as above.